modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/HaskellServantCodegen.java

```diff
@@ -432,11 +431,11 @@ private String makeQueryListType(String type, String collectionFormat) {
             case "pipes": return "(QueryList 'PipeSeparated (" + type + "))";
             case "multi": return "(QueryList 'MultiParamArray (" + type + "))";
             default:
-                throw new NotImplementedException();
+                throw new UnsupportedOperationException();
```

Comment:
I suggest we keep `NotImplementedException` as existing programs may be a dependency on this. If we switch it to `UnsupportedOperationException`, we'll need to flag this change as breaking change, which seems not necessary.